rpa-builder/6.0/modules/ROOT/pages/workbench-toolbar-overview.adoc

```diff
@@ -0,0 +1,213 @@
+* (image:arrow-rotate-right-icon.png[The Redo icon, 2%, 2%]) <<undo-redo, Redo action>>
+* (image:arrow-up.png[The Move selected object up icon, 2%, 2%]) <<move-up-down, Move selected object up>>
+* (image:arrow-down.png[The Move selected object down icon, 2%, 2%]) <<move-up-down, Move selected object down>>
+* (image:delete-icon-alt.png[The Delete selected object icon, 2%, 2%]) <<delete, Delete selected object(s)>>
+* (image:run-wizard-icon.png[The Run wizard for selected object icon, 2%, 2%]) <<start-wizard, Run wizard for selected object>>
+* (image:quick-creation-icon.png[The Start Quick Creation Mode icon, 2%, 2%]) <<start-quick-creation, Start Quick Creation Mode>>
+* (image:validate-icon.png[The Validate Workflow icon, 2%, 2%]) <<validate, Validate Workflow>>
+* (image:play-icon.png[The Run Workflow icon, 2%, 2%]) <<run-workflow, Run Workflow>>
+* (image:debug-icon.png[The Debug Workflow icon, 2%, 2%]) <<debug-workflow, Debug Workflow>>
+* (image:print-icon.png[The Print icon, 2%, 2%]) <<print-workflow, Print>>
+* (image:wrench-icon.png[The Tools icon, 2%, 2%]) <<tools, Tools>>
+
+In addition to the Workbench Toolbar, you can find the <<search, Search>> bar at the end of the Workbench:
+
+image:search-bar.png[The Search bar, 40%, 40%]
+
+[[copy-paste]]
+== Copy and Paste Workflow Object(s)
+Pressing *Copy* (image:copy-icon.png[The copy icon, 2%, 2%]) copies the selected Workflow object(s) and all its underlying elements into the clipboard. Select the target object and press *Paste* (image:paste-icon.png[The paste icon, 2%, 2%]) to paste it. You can also use _Ctrl + C_ for copying and _Ctrl + V_ for pasting.
+
+[[undo-redo]]
+== Undo and Redo Previous Action
+The *Undo* button (image:arrow-rotate-left-icon.png[The copy icon, 2%, 2%]) enables you to undo any previous action you carried out in RPA Builder. If you accidently went back too far, use the *Redo* button (image:arrow-rotate-right-icon.png[The copy icon, 2%, 2%]) to get back to the desired action.
+Using _Ctrl + Z_ for undo and _Ctrl + Y_ for redo also works.
+
+[[move-up-down]]
+== Change the Order of Workflow Objects
+To change the order of objects, select the object and move it up or down in the Workflow by clicking *Move up* (image:arrow-up.png[The Move selected object up icon, 2%, 2%]) or *Move down* (image:arrow-down.png[The Move selected object down icon, 2%, 2%]) until the object reaches the desired position.
+Action Steps cannot be moved outside of their Transaction. If you want to move an Action Step to another Transaction, use copy and paste instead, and then delete the old one. You can also use _Ctrl + X_ or the *Cut* option from the context menu to remove the Action Step from the Workflow and put it into the clipboard in one step.
+
+[[delete]]
+== Delete Workflow Objects
+To delete an object, select it and click *Delete* (image:delete-icon-alt.png[The Delete selected object icon, 2%, 2%]). Pressing the _Del_ key has the same effect.
```

Comment:
I guess the RPA docs use "Press" and not "Click" for a key. I will stop marking them.